evidence/gui/0001-evidence.md

```diff
@@ -0,0 +1,70 @@
+# Evidence-0001: Promotion to Rank 1
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission (2025/01/09)                                                             |
+| **Submitted by**| Guillaume Thiolliere                                                                        |
+
+
+## Member details
+
+- Matrix username: @guillaume:parity.io
+- Polkadot address: 13psJuWEjBuZGaFqXvFnLMC6ME8RVVfQAhtFhydYjW45oKgZ
```

Comment:
You need to set up on-chain identity in order to be inducted and promoted.
I think this guide should still work: https://hackmd.io/@andresilva/rJ8fGMtf6

I opened https://github.com/polkadot-fellows/Evaluations/issues/112 for getting this info and guide in the official docs.

Reply:
Thanks, I set it up, but polkaidentity doesn't seem to support matrix from specific host (parity.io).

I hope the current fields are enough.